src/main/frontend/dicts.cljs

```diff
@@ -214,31 +214,32 @@
         :content/open-in-sidebar "Open in sidebar"
         :content/copy-as-json "Copy as JSON"
         :content/click-to-edit "Click to edit"
-        :settings-page/edit-config-edn "Edit config.edn for current graph"
+        :settings-page/edit-config-edn "Edit config.edn"
+        :settings-page/edit-custom-css "Edit custom.css"
         :settings-page/show-brackets "Show brackets"
         :settings-page/spell-checker "Spell checker"
         :settings-page/auto-updater "Auto updater"
         :settings-page/disable-sentry "Send usage data and diagnostics to Logseq"
-        :settings-page/preferred-outdenting "Enable logical outdenting"
+        :settings-page/preferred-outdenting "Logical outdenting"
         :settings-page/custom-date-format "Preferred date format"
         :settings-page/preferred-file-format "Preferred file format"
         :settings-page/preferred-workflow "Preferred workflow"
-        :settings-page/enable-timetracking "Enable timetracking"
-        :settings-page/enable-tooltip "Enable tooltips"
         :settings-page/enable-shortcut-tooltip "Enable shortcut tooltip"
-        :settings-page/enable-journals "Enable journals"
-        :settings-page/enable-all-pages-public "Enable all pages public when publishing"
-        :settings-page/enable-encryption "Enable encryption feature"
+        :settings-page/enable-timetracking "Timetracking"
+        :settings-page/enable-tooltip "Tooltips"
+        :settings-page/enable-journals "Journals"
+        :settings-page/enable-all-pages-public "All pages public when publishing"
+        :settings-page/enable-encryption "Encryption"
```

Comment:
My personal opinions are
- when it is an input or select control, use a noun phrase
- when it is a toggle control or a button, use verb phrases

like:
- Enable Timetracking
- Show Tooltips, actually it should be a "Show feature preview tooltip"
- Show shortcut tooltips
- Enable Journals
- Make all pages public when publishing
- Edit xxx Settings (for button)
- Enable developer mode
- ....

Reply:
I prefer a more concise label because it makes it easier to skim, but I don't feel strongly about it.

Is this request a blocking change? If so, I'm happy to change it!

Reply:
That's OK.😁
Renaming should be kept in syncing with [Doc](https://github.com/logseq/docs/blob/master/pages/Basic%20settings.md)